build.sbt

```diff
@@ -20,6 +20,7 @@ lazy val microservice = (project in file("."))
     name := appName,
     scalaVersion := "2.11.11",
     targetJvm := "jvm-1.8",
+    scalacOptions += "-Ypartial-unification",
```

Comment:
Are you still using cats now?
Or did you remove the `OptionT` usage?

Reply:
It didnt work though :(